webhook/tests/test_query_results.py

```diff
@@ -4,6 +4,7 @@
 from models import UserEmail
 from google.cloud import ndb
 
+import main
```

Comment:
If we import main here, maybe we should delete
```python
from main import app
```
Because one can got lost why there are two imports of the same think.